### ISSUE TITLE
Testing for the existence of clusters of data

### ISSUE THREAD

**Issue by ikuznet1** [Upward-Spiral-Science/the-fat-boys#2]
We want to test whether our data clusters. In other words, if we denote k as the number of clusters, we want to test H_0 vs H_a, where:
H_0: k = 1
H_a: k > 1

We plan to do this via the methodology described here: http://www.stat.ufl.edu/archived/casella/Papers/paper-v3.pdf

Does this seem reasonable? Are there other well known methods?

**Comment by jovo** [comment]
this is my favorite way to test for that:

On Thu, Feb 18, 2016 at 3:10 PM, ikuznet1 notifications@github.com wrote:

> We want to test whether our data clusters. In other words, if we denote k
> as the number of clusters, we want to test H_0 vs H_a, where:
> H_0: k = 1
> H_a: k > 1
> 
> We plan to do this via the methodology described here:
> http://www.stat.ufl.edu/archived/casella/Papers/paper-v3.pdf
> 
> Does this seem reasonable? Are there other well known methods?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Upward-Spiral-Science/the-fat-boys/issues/2.

## 

the glass is all full:  half water, half air.
neurodata.io

**Comment by jovo** [comment]
[McLachlan87a.pdf](https://github.com/Upward-Spiral-Science/the-fat-boys/files/136865/McLachlan87a.pdf)

**Comment by jovo** [comment]
R code is in Ch1: https://github.com/jovo/nam

**Comment by ikuznet1** [comment]
I think the provided code is only for univariate data. A similar approach using mclust should work.

**Comment by jovo** [comment]
agreed.  of note, doing it well, we could probably publish a paper on
theory/methods....

On Thu, Feb 18, 2016 at 6:23 PM, ikuznet1 notifications@github.com wrote:

> I think the provided code is only for univariate data. A similar approach
> using mclust should work.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Upward-Spiral-Science/the-fat-boys/issues/2#issuecomment-185977925
> .

## 

the glass is all full:  half water, half air.
neurodata.io

**Comment by ikuznet1** [comment]
Interesting. I'll play around with it for now, but we should meet at some point to figure out what doing it well involves.
